### ISSUE TITLE
Comparison with ortools

### ISSUE THREAD

**Issue by mufassir-khan** [VROOM-Project/vroom#622]
Hi @jcoupey ,
I was trying to compare the performance of vroom solver to the performance of ortools solver. Here are my observations:
1. vroom is way fast in terms of solver run time as compared to ortools.
2. The quality of solutions is better than ortools when number of jobs(max_number_of_tasks) per vehiclen for all vehicles is >= 5. The quality is slightly lower than ortools when number of jobs(max_number_of_tasks) per vehicle for all vehicles are less than 4. 

Do you have any such observations or any comments on this?

Quality of solutions is being checked by finding average inter-job distance in the solutions of vroom and ortools respectively.

**Comment by jcoupey** [comment]
I have no first-hand experience with OR-Tools but I did some testing and comparing based on instances/results provided by other users I've worked with.

The general trend we saw was that VROOM provides solutions that are similar in quality, or even significantly better in some cases, while being an order of magnitude faster. The feedback I often get is that scaling with OR-Tools is much harder, due to the computing times.

This is not a real surprise if you look under the hood: VROOM is dedicated to solving VRP and has a lot of ad-hoc stuff baked in for that purpose, while OR-Tools is an all-purpose generic solving engine that can handle many other kind of problems. On the other hand it means that OR-Tools is much more flexible if you want to add arbitrary constraints to your model.

If you want to push the comparison, a good approach would be to use the usual benchmarks from the academic literature. We have our own [result page](https://github.com/VROOM-Project/vroom/wiki/Benchmarks) but I'm not aware of any such overview for OR-Tools. A starting point with VRPTW benchmarks can be found in [this report](https://www.confer.cz/clc/2019/download/2922-comparison-of-capabilities-of-recent-open-source-tools-for-solving-capacitated-vehicle-routing-problems.pdf) but it probably does not really show the big picture since the comparison is limited to 100-point instances.
